Flipbook/turn.js

```diff
@@ -0,0 +1,1933 @@
+/**
+ * turn.js 3rd release
+ * www.turnjs.com
+ *
+ * Copyright (C) 2012, Emmanuel Garcia.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright
+ * notice, this list of conditions and the following disclaimer.
+ *
+ * 2. Any redistribution, use, or modification is done solely for personal 
+ * benefit and not for any commercial purpose or for monetary gain.
+ * 
+ **/
```

Comment:
@shivam123-dev i don't think u made this from ur own

Reply:
No @arpit456jain sir I didn't made the turn.js API. With the help of this API, I just added the turning animation. This API (turn.js) has been downloaded from its [original source](http://turnjs.com/).

Reply:
So, I request you to kindly remove the label of plagiarism.